packages/driver/cypress/e2e/commands/actions/selectFile.cy.js

```diff
@@ -628,23 +628,30 @@ is being covered by another element:
         })
       })
 
-      it('waits until input stops animating', {
-        defaultCommandTimeout: 1000,
-      }, () => {
-        let retries = 0
+      it('retries until label is not disabled', () => {
```

Comment:
There's no need to test animation specifically - .disabled is part of `$actionability.verify()` as well. Enabling and disabling an element basically tests the same thing, and in general I prefer less-mocked tests over more-mocked tests.